src/scripts/index.js

```diff
@@ -0,0 +1,20 @@
+import createTag from "./utils/createTag.js";
+
+//Send request DOM API elements
+const divSend = createTag({ className: "div-send-request" });
```

Comment:
We want to use proper class names convention. The name with tag name is really problematic, because when we change tag name we will have problem with class consistency. I prefer use BEM, but other class convention will be ok also. This tag is our block, and our component is 'Send', so I think that class 'send ' is better name than others.

Reply:
Ok, corrected using BEM convention.